packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -254,6 +270,112 @@ describe("wrangler", () => {
       `);
     });
 
+    it("should offer to create a worker in a non-typescript project", async () => {
+      mockConfirm(
+        {
+          text: "Would you like to install wrangler into your package.json?",
+          result: false,
+        },
+        {
+          text: "Would you like to use typescript?",
```

Comment:
Nit: Technically unrelated to this PR, but going off my other comment on capitalization. Could this be? 
```suggestion
          text: "Would you like to use TypeScript?",
```